src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -2713,7 +2713,7 @@
 					"-DSQLCIPHER_CRYPTO_CC",
 					"-DNDEBUG",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = de.rki.coronawarnapp;
+				PRODUCT_BUNDLE_IDENTIFIER = de.rki.coronawarnapp-dev;
```

Comment:
are the changes to the project file required, intended or accidental?

Reply:
That's accidental, I think it came through in a merge from a local test branch. Thanks for flagging that, will remove it now.